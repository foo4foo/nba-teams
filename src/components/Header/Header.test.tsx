import React from 'react'
import renderer from 'react-test-renderer'
import { Header } from './'

describe('Header', () => {
  it('renders successfuly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
