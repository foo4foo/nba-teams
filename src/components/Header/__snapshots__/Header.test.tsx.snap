// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders successfuly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 brLKxe"
>
  <button
    className="StyledButton-sc-323bzc-0 jfbKsv"
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    type="button"
  >
    <span
      className="StyledText-sc-1sadyjn-0 jScuyG"
      size="large"
    >
      NBA
    </span>
  </button>
</div>
`;
